demo/api/schema.gql

```diff
@@ -842,7 +842,14 @@ type Mutation {
   updateDamFile(id: ID!, input: UpdateDamFileInput!): DamFile!
   importDamFileByDownload(url: String!, scope: DamScopeInput!, input: UpdateDamFileInput!): DamFile!
   moveDamFiles(fileIds: [ID!]!, targetFolderId: ID): [DamFile!]!
-  copyFilesToScope(fileIds: [ID!]!, inboxFolderId: ID!): CopyFilesResponse!
+  copyFilesToScope(fileIds: [ID!]!, inboxFolderId: ID!): CopyFilesResponse! @deprecated(reason: "Use copyFiles instead")
+  copyFiles(
```

Comment:
Should we call this `copyDamFiles`? All other DAM mutations include Dam in their names

Reply:
Yes you are right